frontend/src/views/c-authorship.vue

```diff
@@ -157,7 +173,23 @@ export default defineComponent({
   emits: [
     'deactivate-tab',
   ],
-  data() {
+  data(): {
+    isLoaded: boolean,
+    selectedFiles: Array<AuthorshipFile>,
+    filterType: FilterType,
+    selectedFileTypes: Array<string>,
+    fileTypes: Array<string>
+    filesLinesObj: {[key: string]: number}
```

Comment:
```suggestion
    filesLinesObj: { [key: string]: number }
```

Reply:
Might be worth looking into implementing a linting rule for the spaces in such types... (in a separate PR)